frontend/src/api/alerts/save.ts

```diff
@@ -0,0 +1,17 @@
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { PayloadProps, Props } from 'types/api/alerts/save';
+
+import create from './create';
+import put from './put';
```

Comment:
Can we remove this one and move this logic in the component itself

Reply:
I think this makes the formAlertRules cleaner and separation of concern. It doesn't need to have multiple api calls. 

Is there any specific issue you see with this?